index.html

```diff
@@ -20,24 +20,165 @@
       </nav>
     </header>
 
-    <!-- The headline section -->
-    <section class="container">
-      <h1>I’m Raymond Glad to see you!</h1>
-      <p>I’m a software developer! I can help you build a product ,
-        feature or website Look through some of my work and experience!
-        If you like what you see and have a project you need coded,
-        don’t hestiate to contact me.
-      </p>
-
-      <!-- Social contact -->
-      <a href="#" class= "connect">LET'S CONNECT</a>
-      <ul class="socials">
-        <li><a class="contacts" href="#"><img src="img/twitter-icon.png" alt='Twitter' /></a></li>
-        <li><a class="contacts" href="#"><img src="img/linkedin-icon.png" alt='linkedIn' /></a></li>
-        <li><a class="contacts" href="#"><img src="img/mail-icon.png" alt="Mail"/></a></li>
-        <li><a class="contacts" href="#"><img src="img/github-icon.png" alt="Github" /></a></li>
-        <li><a class="contacts" href="#"><img src="img/thumbup-icon.png" alt="Thumbup" /></a></li>
-      </ul>
-     </section>
+    <div class= "frame">
+      <!-- The headline section -->
+      <div class="container">
+        <h1>I’m Raymond Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded,
+          don’t hestiate to contact me.
+        </p>
+
+        <!-- Social contact -->
+        <a href="#" class= "connect">LET'S CONNECT</a>
+        <ul class="socials">
+          <li><a class="contacts" href="#"><img src="img/twitter-icon.png" alt='Twitter' /></a></li>
+          <li><a class="contacts" href="#"><img src="img/linkedin-icon.png" alt='linkedIn' /></a></li>
+          <li><a class="contacts" href="#"><img src="img/mail-icon.png" alt="Mail"/></a></li>
+          <li><a class="contacts" href="#"><img src="img/github-icon.png" alt="Github" /></a></li>
+          <li><a class="contacts" href="#"><img src="img/thumbup-icon.png" alt="Thumbup" /></a></li>
+        </ul>
+
+        <!-- Works Section -->
+         <section class="work">
+            <div class="grid-item frame-case-a">
+              <!-- <img src="img/Snapshoot-Portfolio-a.png" alt=""> -->
```

Comment:
Great job @dialite 🤝

- [ ] [Optional] Kindly remove commented code from your project as this is regarded as bad practice.